### DISCUSSION TITLE
Hyperparameter tuning

### DISCUSSION THREAD

**Discussion by siddancha** [mujocolab/mjlab#570] [Q&A]
This question is just out of curiosity. I noticed that the task RL configs share nearly identical PPO hyperparameters (learning rate, gamma, clip param, network architecture) with few per-task differences (entropy coefficient, obs normalization, training length).

Were these defaults tuned, or are they just standard PPO values that work well enough? If tuning was done, do the authors recommend any particular tools or strategies for hyperparameter tuning or grid searches/sweeps in mjlab? Thanks!
